commcare_connect/users/models.py

```diff
@@ -69,3 +69,11 @@ class Role(models.TextChoices):
         related_name="memberships",
     )
     role = models.CharField(max_length=20, choices=Role.choices, default=Role.MEMBER)
+
+
+class ConnectIDUserLink(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+    )
+    commcare_username = models.TextField()
```

Comment:
This could be username or user id. I think both are in the forms so I went with username to match what we do in HQ, but could switch it if that is preferred.